app/models/ca_search_forms.php

```diff
@@ -1145,6 +1183,22 @@ public function getHTMLFormElements($po_request, $pa_form_data=null) {
 						'name' => $va_element['bundle_name']
 					);
 					continue(2);
+				case 'hidden_field':
+					if (!$vs_field_label) { $vs_field_label = ''; }
+					if(isset($va_element['settings']['hidden_field_name']) && isset($va_element['settings']['hidden_field_value'])){
```

Comment:
So it does nothing if you don't give it a `name` and a `value`, which seems fair enough, but should there be a warning somewhere?